interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day/ajax_template.html

```diff
@@ -542,6 +544,11 @@
             $eStartMin = $starth * 60 + $startm;
             $dispstarth = ($starth > 12) ? ($starth - 12) : $starth; // used to display the hour
 
+
+             //fix bug 456 and 455
+             //check to see if the event is in the clinic hours range, if not it will not be displayed
+             if  ((int)$starth >= (int)$openhour && (int)$starth<(int)$closehour ){
```

Comment:
I think would be more clear to use a continue to skip to the next item:
``````
//check to see if the event is in the clinic hours range, if not it will not be displayed
if (conditional showing the appt is outside the clinic hours) continue;